src/js/page_sections/Header.js

```diff
@@ -1,5 +1,7 @@
 import React, { Component } from 'react';
 import { defineMessages, injectIntl } from 'react-intl';
+import { withSiteData } from 'react-static';
```

Comment:
I can't remember how we landed on moving this up to Header. But I'm thinking all the siteData 🎶 and 💃  should be done in LanguageWrapper and passed into Header (or whatever sub component may need it). This way, if you want to make a storybook out of header too.